docs/api/structures/process-metric.md

```diff
@@ -7,6 +7,7 @@
     The time is represented as number of milliseconds since epoch.
     Since the `pid` can be reused after a process dies,
     it is useful to use both the `pid` and the `creationTime` to uniquely identify a process.
+* `memory` [MemoryInfo](memory-info.md) (optional) _macOS_ _Windows_ - Memory information for the process.
```

Comment:
http://nadeausoftware.com/articles/2012/07/c_c_tip_how_get_process_resident_set_size_physical_memory_use has sample code on how to find `workingSetSize` and `peakWorkingSetSize` on Linux -- `/proc/self/statm` and getrusage(), respectively

Reply:
I will have a look when I setup an Ubuntu VM build machine